### ISSUE TITLE
Please stop spamming me

### ISSUE THREAD

**Issue by asmeurer** [regro/releases#1]
This is the second time in a week I've been automatically added to a repo then spammed with a thousand email notifications.

**Comment by beckermr** [comment]
this is a setting in your github settings.

**Comment by beckermr** [comment]
I would love to remove you but I cannot.

**Comment by beckermr** [comment]
See here: 
<img width="1151" alt="Screen Shot 2020-08-18 at 5 51 49 PM" src="https://user-images.githubusercontent.com/5296416/90573221-8792e700-e17b-11ea-8332-9a49fa9758a7.png">

**Comment by beckermr** [comment]
uncheck automatically watching

**Comment by beckermr** [comment]
cc @scopatz in case he wants to not be spammed too!

**Comment by asmeurer** [comment]
I think I'm going to just remove myself from this org. I'm not really involved in any of the repos on it.

**Comment by beckermr** [comment]
sounds good! my apologies for the spam!

**Comment by beckermr** [comment]
feel free to close this issue when you are ready!

**Comment by scopatz** [comment]
I unwatched this repo FWIW

**Comment by asmeurer** [comment]
Yeah I also unwatched, but then @beckermr just made another repo that I was automatically added to.
